pom.xml

```diff
@@ -247,6 +255,278 @@
                                     org.wso2.carbon.registry:org.wso2.carbon.registry.contentsearch.feature:${carbon.registry.version.4.4.4}
                                 </featureArtifactDef>
 
+				<!-- ESB Features -->
```

Comment:
Instead of mention the features as Product specific, is it ok to have the features alphabetical order, i.e., all commons together since there can be different version of the components, this helps readability.